README.md

```diff
@@ -1,23 +1,25 @@
-# InfluxData Helm Charts
+# InfluxData Helm charts
 
 [![License](https://img.shields.io/badge/license-MIT-green.svg)](./LICENSE)
 [![](https://github.com/influxdata/helm-charts/workflows/helm-charts%2Frelease/badge.svg?branch=master)](https://github.com/influxdata/helm-charts/actions)
 
-
-This functionality is in beta and is subject to change. The code is provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+InfluxData Helm charts are in beta and subject to change. The code is provided as-is with no warranties. Beta features are not subject to the support service level agreement (SLA) of general availability (GA) features.
 
 ## Usage
 
-[Helm](https://helm.sh) must be installed to use the charts.
-Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+1. Install [Helm](https://helm.sh). For more information, see [Helm documentation](https://helm.sh/docs/).
+
+2. Add the InfluxData Helm repository:
 
-Once Helm is set up properly, add the repo as follows:
+    ```console
+        helm repo add influxdata https://helm.influxdata.com/
```

Comment:
Don't think this should be indented

Reply:
We align code blocks with the text on the line above, so looks like 2 instead of 3:

![image](https://user-images.githubusercontent.com/47797004/81124989-b9917d80-8eeb-11ea-9c8f-6124444d3855.png)

Reply:
I get that, but I don't know why the code inside the code block has an additional indent?

Reply:
Thank you, @rawkode -- see your point. Will try tweaking the alignment tomorrow

Reply:
fixed